### DISCUSSION TITLE
Client TLS handshake failed. The client does not trust the proxy's certificate for selfcare.dialog.lk (OpenSSL Error([('SSL routines', '', 'sslv3 alert certificate unknown')]))

### DISCUSSION THREAD

**Discussion by randomuser9912** [mitmproxy/mitmproxy#6226] [General]
Client TLS handshake failed. The client does not trust the proxy's certificate for selfcare.dialog.lk (OpenSSL Error([('SSL routines', '', 'sslv3 alert certificate unknown')]))

how do i fix this

**Comment by Oleksandr121212** [comment]
i have same problem
